models/articles.models.js

```diff
@@ -21,14 +22,28 @@ exports.selectAllArticles = () => {
 };
 
 exports.selectCommentsByArticleId = (article_id) => {
-  return db
-    .query(
+  const queryProms = [];
+  queryProms.push(
+    db.query(
       "SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC",
       [article_id]
     )
+  );
+  queryProms.push(checkIdExists("articles", "article_id", article_id));
+
+  return Promise.all(queryProms).then((promResults) => {
+    return promResults[0].rows;
+  });
+};
+
+exports.insertNewCommentOnArticle = (author, body, article_id) => {
+  const createdTime = new Date();
```

Comment:
This isn't really necessary. If you have a look at the seed file the table created already has this default behaviour.